app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
```

Comment:
I think probably best to stay consistent with the other project and just stick to the well supported elements. Could you wrap it in a `.well` too per the summary document viewer?